ocs_ci/ocs/platform_nodes.py

```diff
@@ -2247,3 +2249,58 @@ def start_nodes(self, nodes, timeout=600, wait=True):
             wait_for_nodes_status(
                 node_names=node_names, status=constants.NODE_READY, timeout=timeout
             )
+
+
+class IBMCloud(NodesBase):
+    """
+    IBM Cloud class
+    """
+
+    def __init__(self):
+        super(IBMCloud, self).__init__()
+        self.ibmcloud = ibmcloud.IBMCloud()
+
+    def restart_nodes(self, nodes, timeout=900, wait=True):
+        """
+        Restart ibmcloud vm instances by stop and start
+
+        Args:
+            nodes (list): The OCS objects of the nodes / Azure Vm instance
+            timeout (int): time in seconds to wait for node to reach 'not ready' state,
+                and 'ready' state.
+            wait (bool): True if need to wait till the restarted node reaches
+                READY state. False otherwise
+
+        """
+        self.ibmcloud.restart_nodes(nodes, timeout=900, wait=True)
+
+    def restart_nodes_by_stop_and_start(self, nodes, force=True):
+        """
+        Make sure all Nodes are up
+
+        """
+        self.ibmcloud.restart_nodes_by_stop_and_start(nodes, force=True)
```

Comment:
The docstring for `restart_nodes` method  states _Restart ibmcloud vm instances by stop and start_ but then there is the second method `restart_nodes_by_stop_and_start` which have docstring _Make sure all Nodes are up_. I don't know the internals of `ibmcloud.restart_nodes` and `ibmcloud.restart_nodes_by_stop_and_start`, so maybe it is perfectly ok, but for the first look it is slightly confusing...

Reply:
Changed the doctoring